libs/design/src/atoms/form/form-field/form-field/form-field.component.html

```diff
@@ -1,2 +1,7 @@
-<ng-content select="[daff-input]"></ng-content>
+<div class="daff-form-field__control daff-form-field__control--{{ _control.controlType }}" [class.daff-error]="isError" [class.daff-valid]="isValid">
```

Comment:
what's the purpose of the `daff-form-field__control--{{ _control.controlType }}`? Is that for something in the future? Do we need it now?

Reply:
This is to add only the chevron icon to `daff-native-select` elements.

Reply:
No, that's what `*ngIf="_control.controlType=='native-select'"` does on line 3. I'm pretty sure this class doesn't do anything

Reply:
I think I should've moved the `&__icon` styles into the `native-select` variant, but I don't know if that really makes a difference in terms of making sure that the icon isn't styled or showing up on any elements but the native select.

Reply:
I think the `&__icon` styles should be here. I'm just talking about the `daff-form-field__control--{{ _control.controlType }}` class

Reply:
@lderrickable removed, as you're correct, it did nothing. 74d7ebb3d2e3cfe08c03b5ff3e797936c29fcb9c

Reply:
`daff-form-field__control--{{ _control.controlType }}` adds the class `daff-form-field__control--native-select` if it's a native select element. There's no styles right now because `&__icon` is not specified under `daff-form-field__control--native-select`.

You want to remove `daff-form-field__control--native-select` right? My comment was regarding if it really makes a difference if the `&__icon` styles is inside `daff-form-field__control--native-select` or not.

So:

```.daff-form-field {
  &__control {
    &--native-select {
      &__icon {}
    }
  }
}```

vs

```.daff-form-field {
  &__control {
    &__icon {}
  }
}```

My thought is that if I move `&__icon` into `daff-form-field__control--native-select`, it will ensure that the `input` element will not show `&__icon`.

Reply:
nvm, @damienwebdev removed it 74d7ebb